src/common/config_validator.cpp

```diff
@@ -143,11 +154,7 @@ void ConfigValidator::validateDevices(cli::mode mode) const {
   }
 
   UTIL_THROW_IF2(!regex::regex_match(devices, pattern),
-                 "the argument '(" + devices
-                     + ")' for option '--devices' is invalid. "
-                     + help);
-
-
+                 "the argument '(" + devices + ")' for option '--devices' is invalid. " + help);
```

Comment:
THROW probably copied from here. Is there any reason these should be exceptions?